index.js

```diff
@@ -1,83 +1,59 @@
-/* globals Symbol: false, Uint8Array: false, WeakMap: false */
-/*!
- * deep-eql
- * Copyright(c) 2013 Jake Luer <jake@alogicalparadox.com>
- * MIT Licensed
- */
-
-import type from 'type-detect';
-
-function FakeMap() {
-  this._key = 'chai/deep-eql__' + Math.random() + Date.now();
-}
-
-FakeMap.prototype = {
-  get: function get(key) {
-    return key[this._key];
-  },
-  set: function set(key, value) {
-    if (Object.isExtensible(key)) {
-      Object.defineProperty(key, this._key, {
-        value: value,
-        configurable: true,
-      });
-    }
-  },
-};
+import type from "type-detect";
```

Comment:
We can also take the opportunity to drop this dep, as it’ll mostly be used here for compatibility for older browsers.